src/index.ts

```diff
@@ -497,8 +499,23 @@ function formatAggregates(aggregatesProto: AggregatesProto = {}): Aggregates {
       if (obj[item.key] === undefined) {
         obj[item.key] = {};
       }
-      // @ts-ignore
-      obj[item.key][item.type] = item.value;
+
+      let field = obj[item.key][item.type];
+      if (
+        item.type === "count" &&
+        item.key === "buckets" &&
+        typeof field === "object" &&
+        typeof item.value === "object"
+      ) {
+        field = {
```

Comment:
`field` is never read after assigning it. Should we be adding it to `obj` or is the intention to throw it away?